sycl/source/detail/graph_impl.cpp

```diff
@@ -226,6 +226,141 @@ bool graph_impl::clearQueues() {
   return AnyQueuesCleared;
 }
 
+static bool
```

Comment:
Rather than using `static`, we should put these helper functions in an anonymous namespace like the one at the top of the file.